client/auth.go

```diff
@@ -285,10 +285,8 @@ func (c *Conn) writeAuthHandshake() error {
 		return fmt.Errorf("invalid collation name %s", collationName)
 	}
 
-	// the MySQL protocol calls for the collation id to be sent as 1, where only the
```

Comment:
updating this comment which was incorrect regarding the first byte of the 23 bytes of filter being used to store the right 8 bits of the collation id.